src/pexprs-toArgumentNameList.js

```diff
@@ -56,7 +56,13 @@ pexprs.Terminal.prototype.toArgumentNameList = function(firstArgIndex) {
 };
 
 pexprs.Range.prototype.toArgumentNameList = function(firstArgIndex) {
-  return [this.from + '_to_' + this.to];
+  var argName = this.from + '_to_' + this.to;
+  // If the `argName` doesn't start with an valid character, i.e. alphabet or `_`,
+  // then prepending a `_` to it.
```

Comment:
Grammar nits:
- "a valid", not "an valid" 
- "letter", not "alphabet"
- "prepend", not "prepending"